tests/test-cases/Group1-Docker-Commands/1-05-Docker-Start.robot

```diff
@@ -120,3 +120,12 @@ Start a container with removed network
     ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} start ${container}
     Should Be Equal As Integers  ${rc}  1
     Should Contain  ${output}  network test-network not found
+
+Simple start with attach
+    Run And Return Rc And Output  docker %{VCH-PARAMS} pull ${busybox}
+    ${rc}  ${container}=  Run And Return Rc And Output  docker %{VCH-PARAMS} create ${busybox} ls
```

Comment:
naw, I intentionally didn't check it on either command because it is not the purpose of the test.  I only checked the last RC for that reason.